docs/2.guide/4.recipes/4.sessions-and-authentication.md

```diff
@@ -0,0 +1,403 @@
+---
+title: 'Sessions and Authentication'
+description: "User registration and authentication is an extremely common requirement in web apps. This recipe will show you how to implement basic user registration and authentication in you Nuxt app."
+---
+
+
+## Introduction
+
+In this recipe we'll be setting up user registration, login, sessions, and authentication in a full-stack Nuxt app.
+ We'll be using [Nuxt Auth Utils](https://github.com/Atinux/nuxt-auth-utils) by [Altinux (Sébastien Chopin)](https://github.com/Atinux) which provides convenient utilities for managing front-end and back-end session data. We'll install and use this to get the core session management functionality we're going to need to manage user logins. For the database ORM we'll be using [Drizzle](https://orm.drizzle.team/) with [db0](https://db0.unjs.io/), but you can use any ORM or database connection strategy you prefer.
+
+You'll need a `users` table in your database with the following columns:
+- `id` (int, primary key, auto increment)
+- `email` (varchar)
+- `password` (varchar)
+
+Additionally, we'll use [nuxt-aut-utils](https://github.com/Atinux/nuxt-auth-utils) by [Atinux](https://github.com/Atinux) to handle the authentication and session management.
+
+## Steps
+
+### 1. Install nuxt-auth-utils
+
+
+Install the [auth-utils](https://github.com/Atinux/nuxt-auth-utils) module using the `nuxi` CLI.
```

Comment:
Likely no need to repeat this again here, we can probably say:

```md
Install the module using the `nuxi` CLI.
```

Reply:
I think this is important to leave in. This is the first step, and the first action that a person needs to actually perform. People skip introductions all the time, so I wouldn't count on that being a place for instructions.

Reply:
Although I personally believe it would be the reader's fault for skipping chunks of information, if you think it's worth keeping in, we should probably keep it consistent with the other references, so Nuxt Auth Utils.
Though realistically speaking, there's only so much we can do to aid those who skip documentation without hurting those who don't